### PR TITLE
Add latest to db cluster

### PR DESCRIPTION
**Pull request by JeppeV**
Removing the latest value from the api and adding it to the database. This prevents the latest id from reporting incorrect values as a result of the load balancer forwarding subsequent requests to different droplets.

### CODE REVIEW

File: api/db/schema.sql [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
   user_id INTEGER PRIMARY KEY AUTO_INCREMENT,
-  username VARCHAR(255) NOT NULL,
-  email VARCHAR(255) NOT NULL,
-  password VARCHAR(255) NOT NULL
+  username VARCHAR
+(255) NOT NULL,
+  email VARCHAR
+(255) NOT NULL,
+  password VARCHAR
+(255) NOT NULL
```
Comment (Fr3d-) [inline api/db/schema.sql:9]: is this on purpose?

File: api/db/schema.sql [hunk 2]
```diff
@@ -17,7 +20,14 @@ DROP TABLE IF EXISTS messages;
 CREATE TABLE messages (
   message_id INTEGER PRIMARY KEY AUTO_INCREMENT,
   user_id INTEGER NOT NULL,
-  text VARCHAR(255) NOT NULL,
+  text VARCHAR
+(255) NOT NULL,
```
Comment (Fr3d-) [inline api/db/schema.sql:24]: same with this

File: api/db/schema.sql (hunk 1 above)
Comment (JeppeV) [reply api/db/schema.sql:9]: It was auto formatted like this when I made changes to the file.

Comment (Fr3d-) [reply api/db/schema.sql:9]: Unless you have any objections I think the other style is more readable

**Pull request merged**